Formula/gnupg@2.0.rb

```diff
@@ -79,7 +71,17 @@ def post_install
   end
 
   test do
-    Gpg.create_test_key(testpath)
```

Comment:
Yes, the existing system was designed around everything being publicly accessible, for better or worse.